src/cls/IPM/Utils/Module.cls

```diff
@@ -1353,6 +1369,38 @@ ClassMethod LoadNewModule(
     quit tSC
 }
 
+/// Check whether an update from a currently installed module to a new version of the module is a valid operation.
+ClassMethod CheckOperationVersionValidity(
+	ByRef newModule,
```

Comment:
Nit: newModule need not be passed byref (not making changes) and should have a type